src/server/logging/log_interceptor.js

```diff
@@ -0,0 +1,47 @@
+import Stream from 'stream';
+import { get, isEqual, sortBy } from 'lodash';
+
+function doTagsMatch(event, tags) {
+  return isEqual(sortBy(get(event, 'tags')), sortBy(tags));
```

Comment:
Repeating here since my comment is hidden after the last update:

Is there a reason why we need to support not matching the order?